articles/azure-maps/spatial-io-add-ogc-map-layer.md

```diff
@@ -44,30 +45,32 @@ The following sections outline the web map service features that are supported b
 
 The `url` can be the base URL for the service or a full URL with the query for getting the capabilities of the service. Depending on the details provided, the WFS client may try several standard URL formats to determine how to initially access the service.
 
-The following code shows how to overlay an OGC map layer on the map.
+The [OGC map layer] sample shows how to overlay an OGC map layer on the map.
 
-<br/>
-
-<iframe height='700' scrolling='no' title='OGC Map layer example' src='//codepen.io/azuremaps/embed/xxGLZWB/?height=700&theme-id=0&default-tab=js,result&embed-version=2&editable=true' frameborder='no' allowtransparency='true' allowfullscreen='true'>See the Pen <a href='https://codepen.io/azuremaps/pen/xxGLZWB/'>OGC Map layer example</a> by Azure Maps (<a href='https://codepen.io/azuremaps'>@azuremaps</a>) on <a href='https://codepen.io'>CodePen</a>.
-</iframe>
+:::image type="content" source="./media/spatial-io-add-ogc-map-layer/ogc-map-layer.png"alt-text="A screenshot that shows the snap grid on map.":::
 
+<!----------------------------------------------
+<iframe height='700' scrolling='no' title='OGC Map layer example' src='//codepen.io/azuremaps/embed/xxGLZWB/?height=700&theme-id=0&default-tab=js,result&embed-version=2&editable=true' frameborder='no' allowtransparency='true' allowfullscreen='true'>See the Pen <a href='https://codepen.io/azuremaps/pen/xxGLZWB/'>OGC Map layer example</a> by Azure Maps (<a href='https://codepen.io/azuremaps'>@azuremaps</a>) on <a href='https://codepen.io'>CodePen</a>.</iframe>
+---------------------------------------------->
 ## OGC map layer options
 
-The below sample demonstrates the different OGC map layer options. You may click on the code pen button at the top-right corner to edit the code pen.
+The [OGC map layer options] sample demonstrates the different OGC map layer options. You can select the code pen button at the top-right corner to edit the code pen.
```

Comment:
Delete the sentence "You can select the code pen button at the top-right corner to edit the code pen."